integration_tests/tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
+  "extends": "@tsconfig/node22/tsconfig.json",
```

Comment:
Something changed recently that stopped the `npm run typecheck` task (specifically the `tsc -p integration_tests` bit) working. This seems to fix it.